revocation/internal/crl/crl.go

```diff
@@ -153,7 +196,52 @@ func Supported(cert *x509.Certificate) bool {
 	return cert != nil && len(cert.CRLDistributionPoints) > 0
 }
 
-func validate(crl *x509.RevocationList, issuer *x509.Certificate) error {
+func hasFreshestCRL(extensions []pkix.Extension) bool {
+	return slices.ContainsFunc(extensions, func(ext pkix.Extension) bool {
+		return ext.Id.Equal(oidFreshestCRL)
+	})
+}
```

Comment:
So this function checks if there's delta crl in cert, and logic in `revocation/crl/fetcher.go` checks if there's delta crl in base crl.
To improve the code readability, can we combine these two logics into one method and move it to `revocation/crl/fetcher.go`? The input would be extensions, and the output would be `(idx, bool)`.